performance-timeline/idlharness.any.js

```diff
@@ -6,11 +6,16 @@
 
 'use strict';
 
+function cast(i, t) {
```

Comment:
Hmm where is this used?

Reply:
Oh - at first I was casting `PerformanceMark` as `PerformanceEntry`, but that didn't work as intended.